lib/modules/articles-featured-widgets/views/widget.html

```diff
@@ -6,7 +6,8 @@
   <div class="o-container o-container--medium c-article-featured-widget__container">
     <div class="c-article-featured-widget__media">
       <a href="{{ w._article._url }}">
-        {{ image.render(w._article.thumbnail.items[0]._pieces[0]) }}
+        {% set thumbnail = apos.images.first(w._article.thumbnail.items) %}
```

Comment:
Can you just pass `w._article.thumbnail`?

Reply:
You can, it'll dig.

Reply:
I think in most Punk Ave projects we pass the image object and let the helper dig down for the image. Might be good for consistency's sake (and less words).

Reply:
Updated.

Reply:
save the trees.